docs/content/images.md

```diff
@@ -60,11 +60,11 @@ Align images with the [helper float classes]({{ site.baseurl }}/components/utili
 {% endhighlight %}
 
 <div class="bd-example bd-example-images">
-  <img data-src="holder.js/200x200" class="img-rounded center-block" alt="A generic square placeholder image with rounded corners">
```

Comment:
The uses of `.center-block` need to be replaced with `.m-x-auto` (and adding `.display-block` might be necessary).